### PR TITLE
[templates/nextjs-xmcloud] Add xmcloud-specific comments in api/editing/render

### PR DESCRIPTION
**Pull request by art-alexeyenko**
- [x] This PR follows the [Contribution Guide](https://github.com/Sitecore/jss/blob/dev/CONTRIBUTING.md)
- [ ] Changelog updated
- [ ] Upgrade guide entry added

## Description / Motivation
Updates the comment in api/editing/render endpoint for XMCloud apps with description of new Metadata editing mode flow. 
create-sitecore-jss helper cannot determine the addons being installed, so the comment was added as part of an override file in nextjs-xmcloud.

## Testing Details
- [ ] Unit Test Added
- [ ] Manual Test/Other (Please elaborate)

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Chore

### CODE REVIEW

File: packages/create-sitecore-jss/src/templates/nextjs-xmcloud/src/pages/api/editing/render.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+// Bump body size limit (1mb by default) and disable response limit for Sitecore editor payloads
+// See https://nextjs.org/docs/api-routes/request-helpers#custom-config
+export const config = {
+  api: {
+    bodyParser: {
+      sizeLimit: '2mb',
+    },
+    responseLimit: false,
+  },
+};
```
Comment (ambrauer) [inline packages/create-sitecore-jss/src/templates/nextjs-xmcloud/src/pages/api/editing/render.ts:41]: Actually, this is for Chromes only as well (could add the same "[Chromes mode only]" to the comment)

File: packages/create-sitecore-jss/src/templates/nextjs-xmcloud/src/pages/api/editing/render.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import { EditingRenderMiddleware } from '@sitecore-jss/sitecore-jss-nextjs/editing';
+
+/**
+ * This Next.js API route is used to handle GET and POST requests from Sitecore editors.
+ * GET requests are used with Metadata editing mode while POST ones are used with Chromes(legacy) mode.
+ * This route should match the `serverSideRenderingEngineEndpointUrl` in your Sitecore configuration,
+ * which is set to "http://localhost:3000/api/editing/render" by default (see \sitecore\config\<%- appName %>.config).
```
[the remark this answers is not in the document]
Comment (art-alexeyenko) [reply packages/create-sitecore-jss/src/templates/nextjs-xmcloud/src/pages/api/editing/render.ts:7]: Let me rephrase it for XMCloud instead
